File: Programación/Tema 10 - Control y Manejo de Excepciones/AppInstituto2/src/Persona.java
public class Persona {
    private String dni;
    private String nombre;
    private int edad;
    private String direccion;

    public Persona(String dni, String nombre, int edad, String direccion) {
        this.dni = dni;
        this.nombre = nombre;
        this.edad = edad;
        this.direccion = direccion;
    }

    public Persona() {
        this.dni = "3946407T";
        this.nombre = "";
    }
}

File: Contornos Cliente/Examen 2ª Evaluacion/js/Clases/Persona.js
// Crea la clase Persona que hereda de Mamífero
class Persona extends Mamifero {
    constructor(nombre, edad) {
        super();
        this.nombre = nombre;
        this.edad = edad;
    }

    getNombre() { return this.nombre; }
    setNombre(nombre) { this.nombre = nombre; }

    getEdad() { return this.edad; }
    setPrecio(edad) { this.edad = edad; }

    comer() {
        return "Las personas son seres omnivoros";
    }
}